lib/pages/profile_page/profile_page.dart

```diff
@@ -16,20 +20,23 @@ class ProfilePage extends StatefulWidget {
 }
 
 class _ProfileState extends State<ProfilePage> {
+  static String? imgPath = UserController.currentUser?.userPhoto;
+  final File _img = File(imgPath!);
```

Comment:
Don't give imagePath!
The userProfile is optional for now so it might not be present 

Just be careful as the image can be updated in ProfleImageViewer
Don't allow the user to do so until he is in editing mode
Maybe use a readOnly flag there